README.md

```diff
@@ -137,7 +137,46 @@ sudo scope launch --service-token=<token>
 ```
 
 
-## Developing
+## <a name="using-weave-scope-with-kubernetes"></a>Using Weave Scope with Kubernetes
+
+To try scope's Kubernetes integration, we will boot the probe with the
+special `--probe.kubernetes true` flag. While the probe needs to be
+installed on all nodes (master and minions), this flag should only be
+enabled on the Kubernetes master node.
+
+To start we should install scope on all nodes, as shown in [Getting
```

Comment:
As per the normal requirements, you will need to run Scope on every machine you want to monitor, as shown in ...